ketchup/main.py

```diff
@@ -1,43 +1,99 @@
+import datetime
+
+import dateparser
 import datetime as dt
 from ketchup.luffy import LUFFY
-import ketchup.day_counter as dc
+import ketchup.watch_schedule_builder as dc
```

Comment:
I don't know if it's just me, but the alias for the watch_schedule_builder file, felt appropriate to summarise as "wsb" rather than "dc" for when it was the day_counter file which was being imported instead. Let me know what you think about that.

Reply:
fixed!